practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>"<strong>Incorporating Side Information" -&gt; "Incorporating Background Information"</strong></p><p> </p><p><strong>"</strong>For more in-depth understanding of GCN" -&gt; "For more in-depth understanding of the GCN architecture"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>